src/web/routes/application/confirm.js

```diff
@@ -1,12 +1,22 @@
-const pageContent = {
-  title: 'Application successful',
-  heading: 'Application successful'
+const { isNilOrEmpty } = require('./common/predicates')
+
+const getLanguageBase = (language) => {
+  if (isNilOrEmpty(language)) {
```

Comment:
I don' think we should assume that we always get a language: https://stackoverflow.com/a/41738594

Reply:
Yes, you are right, the language header is optional in the request, but the package we use to deal with languages has a default language set (in `internationalisation.js`) of 'en' so if none is provided, then that default will be used, so by the time we get to the confirm page, we will have a language in the request object.